jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/timer/JmixTimer.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.component.timer;
```

Comment:
The component is placed in the `flowui` module, but its js file in the `flowui-kit`. Since this component doesn't have any Spring dependant functionality, you can place it in the `flowui-kit` module either.